stories/Form.js

```diff
@@ -115,7 +115,7 @@ stories
       labelMultiselect={text('MultiSelect', '*multiselect label*')}
       onSelect={action('selected')}
       placeholder={text('Placeholder', 'Tis a placeholder')}
-      size={select('Size', ['small', 'medium', 'large'], 'text')}
+      size={select('Size', ['', 'small', 'medium', 'large'], '') || undefined}
```

Comment:
fyi there is a better way to do this once we upgrade `storybook`, see [here](https://github.com/storybooks/storybook/tree/master/addons/knobs#selectv2)